src/Umbraco.Forms.Integrations.Crm.ActiveCampaign/Api/Management/Controllers/Forms/FormControllerBase.cs

```diff
@@ -0,0 +1,26 @@
+﻿using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Umbraco.Cms.Web.Common.Authorization;
+using Umbraco.Cms.Web.Common.Routing;
+using Umbraco.Forms.Core.Services;
+
+namespace Umbraco.Forms.Integrations.Crm.ActiveCampaign.Api.Management.Controllers.Forms
+{
+    [BackOfficeRoute($"{Constants.ManagementApi.RootPath}/v{{version:apiVersion}}/forms")]
+    [ApiExplorerSettings(GroupName = Constants.ManagementApi.AccountsGroupName)]
```

Comment:
Will update this to another group name, but I've been thinking: should I make this endpoint private, so it won't be displayed on SwaggerUI, because this endpoint only use to retrieve form fields only, and user probably won't use it, since it has nothing to do with ActiveCampaign.

Reply:
As a controller it needs to be public, otherwise it won't be callable at all.  You can hide it from Swagger if you like though (but I don't think it does any harm to be available) - see https://code-maze.com/aspnetcore-how-to-hide-endpoint-in-swagger/ under "Using ApiExplorerSettings Attribute With the IgnoreApi Parameter".